### ISSUE TITLE
Problems with suda_smart_edit if it's set after the plugin loaded. [[SOLVED]]

### ISSUE THREAD

**Issue by marcg03** [lambdalisue/vim-suda#68]
I use neovim with [lazy.nvim](https://github.com/folke/lazy.nvim) and I initially installed **suda.vim** like this: `{ "lambdalisue/suda.vim" }` .  
When I tried to use the `suda_smart_edit` functionality by adding `vim.g.suda_smart_edit = 1` to my _~/.config/nvim/after/plugin_ folder the plugin didn't work.  
  
Then I realized I had to set `vim.g.suda_smart_edit` before loading the plugin so I changed my installation to this:
```lua
{
    "lambdalisue/suda.vim",
    init = function()
        vim.g.suda_smart_edit = 1
    end
}
```

# TL; DR
Set the **suda_smart_edit** global variable before loading **suda.vim** if you want smart edit functionality.
